### PR TITLE
fix: improve tolerance for error handling multivariate anomaly detection

### PR DESCRIPTION
**Pull request by eisber**
## Related Issues/PRs

Improve error handling for multivariate anomaly detection 

## How is this patch tested?

- [ ] I have written tests (not required for typo or doc fix) and confirmed the proposed feature/bug-fix/change works.

## Does this PR change any dependencies?

- [x] No. You can skip this section.
- [ ] Yes. Make sure the dependencies are resolved correctly, and list changes here.

## Does this PR add a new feature? If so, have you added samples on website?

- [x] No. You can skip this section.
- [ ] Yes. Make sure you have added samples following below steps.

### CODE REVIEW

File: cognitive/src/main/scala/com/microsoft/azure/synapse/ml/cognitive/anomaly/MultivariateAnomalyDetectorSchemas.scala [hunk 1]
```diff
@@ -28,7 +28,7 @@ case class DMASummary(status: String,
 
 object DMAError extends SparkBindings[DMAError]
 
-case class DMAError(code: String, message: String)
+case class DMAError(code: Option[String], message: Option[String])
```
Comment (mhamilton723) [inline cognitive/src/main/scala/com/microsoft/azure/synapse/ml/cognitive/anomaly/MultivariateAnomalyDetectorSchemas.scala:31]: if this is unused can you remove?

Comment (eisber) [reply cognitive/src/main/scala/com/microsoft/azure/synapse/ml/cognitive/anomaly/MultivariateAnomalyDetectorSchemas.scala:31]: it's still used in DMASummary

**Pull request merged**